[13:31:53] Info   :  The log of the day was successfully created and initialized.
[13:31:53] Error  :   File "C:/Users/paul9/Desktop/Cours Centrale Marseille/Cours S7/Projet thématique/face_recognition_schedule/main.py", line 5, in <module>     gui = GUI()   File "C:\Users\paul9\Desktop\Cours Centrale Marseille\Cours S7\Projet thématique\face_recognition_schedule\view\gui.py", line 108, in __init__     self.database_visualisation = DatabaseVisualisation(self)   File "C:\Users\paul9\Desktop\Cours Centrale Marseille\Cours S7\Projet thématique\face_recognition_schedule\view\database_visualisation.py", line 54, in __init__     self.init_database_view()   File "C:\Users\paul9\Desktop\Cours Centrale Marseille\Cours S7\Projet thématique\face_recognition_schedule\view\database_visualisation.py", line 153, in init_database_view     self.create_line(id_=id_, first_name=self.gui.database[id_][0], last_name=self.gui.database[id_][1])   File "C:\Users\paul9\Desktop\Cours Centrale Marseille\Cours S7\Projet thématique\face_recognition_schedule\view\database_visualisation.py", line 82, in create_line     self.text_dict[id_][index]["height"] = 1 + (len(elem) // int(elem["width"])) TypeError: string indices must be integers
[13:56:02] Error  :   File "C:/Users/paul9/Desktop/Cours Centrale Marseille/Cours S7/Projet thématique/face_recognition_schedule/main.py", line 5, in <module>     gui = GUI()   File "C:\Users\paul9\Desktop\Cours Centrale Marseille\Cours S7\Projet thématique\face_recognition_schedule\view\gui.py", line 108, in __init__     self.database_visualisation = DatabaseVisualisation(self)   File "C:\Users\paul9\Desktop\Cours Centrale Marseille\Cours S7\Projet thématique\face_recognition_schedule\view\database_visualisation.py", line 54, in __init__     self.init_database_view()   File "C:\Users\paul9\Desktop\Cours Centrale Marseille\Cours S7\Projet thématique\face_recognition_schedule\view\database_visualisation.py", line 155, in init_database_view     self.create_line(id_=id_, first_name=self.gui.database[id_][0], last_name=self.gui.database[id_][1])   File "C:\Users\paul9\Desktop\Cours Centrale Marseille\Cours S7\Projet thématique\face_recognition_schedule\view\database_visualisation.py", line 81, in create_line     for index, elem in [last_name, first_name, id_]: ValueError: too many values to unpack (expected 2)
[14:27:21] Error  :   File "C:/Users/paul9/Desktop/Cours Centrale Marseille/Cours S7/Projet thématique/face_recognition_schedule/main.py", line 5, in <module>     gui = GUI()   File "C:\Users\paul9\Desktop\Cours Centrale Marseille\Cours S7\Projet thématique\face_recognition_schedule\view\gui.py", line 108, in __init__     self.database_visualisation = DatabaseVisualisation(self)   File "C:\Users\paul9\Desktop\Cours Centrale Marseille\Cours S7\Projet thématique\face_recognition_schedule\view\database_visualisation.py", line 54, in __init__     self.init_database_view()   File "C:\Users\paul9\Desktop\Cours Centrale Marseille\Cours S7\Projet thématique\face_recognition_schedule\view\database_visualisation.py", line 171, in init_database_view     self.create_line(id_=id_, first_name=self.gui.database[id_][0], last_name=self.gui.database[id_][1])   File "C:\Users\paul9\Desktop\Cours Centrale Marseille\Cours S7\Projet thématique\face_recognition_schedule\view\database_visualisation.py", line 96, in create_line     command=self.cancel_modification())]   File "C:\Users\paul9\Desktop\Cours Centrale Marseille\Cours S7\Projet thématique\face_recognition_schedule\view\database_visualisation.py", line 166, in cancel_modification     self.modify_data(self.active_modification_line) AttributeError: 'DatabaseVisualisation' object has no attribute 'active_modification_line'
[14:31:39] Info   :  L'étudiant.e Paul Wassermann (20190142) a bien été enregistré.e!
[14:32:24] Info   :  L'étudiant.e Alisa Tanaka (20190064) a bien été enregistré.e!
[14:33:32] Info   :  L'étudiant.e Fabien Fouville (20190001) a bien été enregistré.e!
[14:34:13] Info   :  L'étudiant.e Jean-Christ Megnassan (20190002) a bien été enregistré.e!
[14:36:39] Info   :  L'étudiant.e Corentin Lange (20180001) a bien été enregistré.e!
[14:39:07] Info   :  L'étudiant.e Rémi Courbon (20190003) a bien été enregistré.e!
[14:53:57] Info   :  L'étudiant.e Ssala Boubou (11111111) a bien été enregistré.e!
